extension/css/cryptup.css

```diff
@@ -1482,6 +1482,10 @@ td {
   margin-top: 8px;
 }
 
+#pgp_block img {
+  width: 100%;
```

Comment:
I think it should be `max-width` property, as currently even small images are stretched to 100% width.

Reply:
Ah, yes. I missed this one. I have corrected it by using `max-width`. Thanks!